src/main/java/net/coderbot/iris/mixin/AccessorRenderLayers.java

```diff
@@ -0,0 +1,17 @@
+package net.coderbot.iris.mixin;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.render.RenderLayer;
+import net.minecraft.client.render.RenderLayers;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import java.util.Map;
+
+@Mixin(RenderLayers.class)
+public interface AccessorRenderLayers {
```

Comment:
Per my comment in IdMapParser, this system as-is is too fragile. I'm removing it for now.